xarray/tests/test_distributed.py

```diff
@@ -19,35 +18,34 @@
 from dask.distributed import Client, Lock
 from distributed.client import futures_of
 from distributed.utils_test import (  # noqa: F401
+    cleanup,
     cluster,
     gen_cluster,
     loop,
-    cleanup,
     loop_in_thread,
 )
 
 import xarray as xr
 from xarray.backends.locks import HDF5_LOCK, CombinedLock
-from xarray.tests.test_backends import (
-    ON_WINDOWS,
-    create_tmp_file,
-    create_tmp_geotiff,
-    open_example_dataset,
-)
-from xarray.tests.test_dataset import create_test_data
-
 from xarray.tests import (
     assert_allclose,
     assert_identical,
     has_h5netcdf,
     has_netCDF4,
-    requires_rasterio,
     has_scipy,
-    requires_zarr,
     requires_cfgrib,
     requires_cftime,
     requires_netCDF4,
+    requires_rasterio,
+    requires_zarr,
+)
+from xarray.tests.test_backends import (
+    ON_WINDOWS,
+    create_tmp_file,
+    create_tmp_geotiff,
+    open_example_dataset,
 )
+from xarray.tests.test_dataset import create_test_data
 
 # this is to stop isort throwing errors. May have been easier to just use
```

Comment:
Could delete these comments and move the `da` import up to Line 19 without the `importorskip`

very minor comment though, we can just leave it as-is

Reply:
you got it!